src/hints/expect_revert/tests.rs

```diff
@@ -1,11 +1,15 @@
-use crate::cli::commands::test::tests::run_single_test;
+use crate::cli::commands::test::{tests::run_single_test, TestCommandError};
 use rstest::rstest;
 
 #[rstest]
 #[case("src/hints/expect_revert/failing.cairo", false)]
 #[case("src/hints/expect_revert/success.cairo", true)]
-fn test_expect_revert(#[case] path: &str, #[case] expected_success: bool) {
+fn test_expect_revert(
+	#[case] path: &str,
+	#[case] expected_success: bool,
+) -> Result<(), TestCommandError> {
 	let path = std::path::PathBuf::from(path);
-	let (_, actual_success) = run_single_test("test_expect_revert", &path);
+	let (_, actual_success) = run_single_test("test_expect_revert", &path)?;
 	assert_eq!(expected_success, actual_success);
+	Ok(())
```

Comment:
Why do you need to return an error?
In test we usually just panic when something does not go according to the plan. And it tag the test as FAILED.

`let (_, actual_success) = run_single_test("test_expect_revert", &path).expect("`run_single_test` should be Ok");`